streams/src/test/java/org/apache/kafka/streams/processor/internals/RepartitionTopicsTest.java

```diff
@@ -447,13 +414,9 @@ private void setupClusterWithMissingTopicsAndMissingPartitionCounts(final Set<St
             SOME_OTHER_TOPIC
         );
         topics.removeAll(missingTopics);
-        expect(clusterMetadata.topics()).andStubReturn(topics);
-        expect(clusterMetadata.partitionCountForTopic(SOURCE_TOPIC_NAME1))
-            .andStubReturn(topicsWithMissingPartitionCounts.contains(SOURCE_TOPIC_NAME1) ? null : 3);
-        expect(clusterMetadata.partitionCountForTopic(SOURCE_TOPIC_NAME2))
```

Comment:
This and the expectation below were not exercised. I ran the tests with PowerMock/EasyMock and without these two expectations and they still passed.